metricbeat/module/docker/helper.go

```diff
@@ -35,37 +51,25 @@ func ExtractContainerName(names []string) string {
 	}
 	return strings.Trim(output, "/")
 }
+
 func BuildLabelArray(labels map[string]string) []common.MapStr {
 
-	output_labels := make([]common.MapStr, len(labels))
+	outputLabels := make([]common.MapStr, len(labels))
 	i := 0
 	var keys []string
 	for key := range labels {
 		keys = append(keys, key)
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
+		// Replace all . in the labels by _
+		// TODO: WHY?
```

Comment:
It was the naming used in dockbeat Project, where it has been decided to format the labels output . I didn't want it to be different, that's why I left it at it is. It's not necessery. We can leave it as it's generated by docker's API.